content/en/docs/languages/java/getting-started.md

```diff
@@ -148,9 +148,14 @@ number of ways, the steps below use environment variables.
    export JAVA_TOOL_OPTIONS="-javaagent:PATH/TO/opentelemetry-javaagent.jar" \
      OTEL_TRACES_EXPORTER=logging \
      OTEL_METRICS_EXPORTER=logging \
-     OTEL_LOGS_EXPORTER=logging
+     OTEL_LOGS_EXPORTER=logging \
+     OTEL_METRIC_EXPORT_INTERVAL=5000
    ```
 
+   Note that `OTEL_METRIC_EXPORT_INTERVAL=5000` (milliseconds) is a temporary
+   setting to test that your metrics are properly recorded. Remember to remove
```

Comment:
@cartermp et al.: is there a better word here than "recorded"?
(Note that "recorded" is also used in the other file addition.)

Reply:
I would prefer the phrase, "properly generated".

Reply:
That works better for me too! @eplusx can you make the change here and in the next file too?

Reply:
Thanks! Updated.